gulp/scripts.js

```diff
@@ -100,7 +100,7 @@ export function lint() {
   // either the tests or app code fail to throw lint errors
```

Comment:
@jonnyshaw89 What does this lint function get used for now ?

Reply:
https://github.com/ONSdigital/eq-survey-runner/pull/1407/files#diff-ba210a9157252cc983268fdc3aa3ed52R116